lib/admin/pm_check.dart

```diff
@@ -148,23 +148,24 @@ class _PmCheckPageState extends State<PmCheckPage> {
                     child: Consumer<TemplateNotifier>(
                         builder: (context, value, child) {
                       return ListView(
-                        children: buildPMList(value),
+                        children: buildPMList(value, context),
                       );
                     })),
-                const VerticalDivider(
+                VerticalDivider(
                   width: 20,
                   thickness: 1,
                   indent: 5,
                   endIndent: 5,
-                  color: Colors.grey,
+                  color: Theme.of(context).dividerColor,
                 ),
                 const Expanded(child: PMDetailView()),
-                const VerticalDivider(
+                VerticalDivider(
+                  //why do we need a divider here? (Dustin)
```

Comment:
having some space looks nicer since the textbox would be right at the edge otherwise.

it's a divider cause i can just copy the other one i had